server/routes/education-support-plan/refuse-plan/validationSchemas/reasonSchema.test.ts

```diff
@@ -20,9 +20,18 @@ describe('reasonSchema', () => {
   })
 
   it.each([
-    { refusalReason: 'EXEMPT_REFUSED_TO_ENGAGE' },
-    { refusalReason: 'EXEMPT_NOT_REQUIRED' },
-    { refusalReason: 'EXEMPT_INACCURATE_IDENTIFICATION' },
+    {
+      refusalReason: 'EXEMPT_REFUSED_TO_ENGAGE',
+      refusalReasonDetails: { EXEMPT_REFUSED_TO_ENGAGE: 'a'.repeat(199) },
+    },
+    {
+      refusalReason: 'EXEMPT_NOT_REQUIRED',
+      refusalReasonDetails: { EXEMPT_NOT_REQUIRED: 'a'.repeat(199) },
+    },
+    {
+      refusalReason: 'EXEMPT_INACCURATE_IDENTIFICATION',
+      refusalReasonDetails: { EXEMPT_INACCURATE_IDENTIFICATION: 'a'.repeat(199) },
+    },
```

Comment:
Is it worth adding a sad path test to this test file? One that sets the refusal reason, but not the refusal reason details, and therefore fails the new mandatory validation?